src/js/components/search/topFilterBar/filterGroups/KeywordFilterGroup.jsx

```diff
@@ -30,16 +30,19 @@ export default class KeywordFilterGroup extends React.Component {
         const tags = [];
 
         // check to see if a keyword is provided
-        const keyword = this.props.filter.values;
+        const keywords = this.props.filter.values;
```

Comment:
This would be a great place to use the [reduce function](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/reduce).